app/views/articles/show.html.erb

```diff
@@ -1,2 +1,77 @@
-<h1>Articles#show</h1>
-<p>Find me in app/views/articles/show.html.erb</p>
+<div class="container">
+  <div class="row">
+
+    <div class="col-xs-8">
+
+      <h2><%= @article.title %></h2>
+      <p><em><%= @article.summary %></em></p>
+      <p>(£ <%= @article.price %> per day) </p>
+
+      <!-- Show URL/content only if accessible, i.e. owned or booked -->
+
+      <% accessible_articles = current_user.articles + current_user.today_booked_articles %>
```

Comment:
The articles accessible to the user at the moment, i.e. the ones they booked, with start_date <= today and end_date >= today